test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+cargo test --lib -- --nocapture --include-ignored ::anchor
+yarn run ts-mocha -p ./tsconfig.json -t 1000000 solana/restaking/tests/**/*.ts
```

Comment:
```suggestion
yarn run ts-mocha -p ./tsconfig.json -t 1000000 solana/restaking/tests/**/*.ts
```

If you want to make it POSIX compliant than:

```sh
#!/bin/sh
cargo test --lib -- --nocapture --include-ignored ::anchor
find solana/restaking/tests/ -name '*.ts' \
     -exec yarn run ts-mocha -p ./tsconfig.json -t 1000000 {} +
```

Either way, I’m not sure if `test.sh` is really the best name for this
file.  It only does Solana tests so `test-solana.sh` seems more
appropriate.

Reply:
yes yes `test-solana.sh` is a better name.